ansible_ai_connect/ai/api/model_pipelines/llamacpp/configuration.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright Red Hat
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from dataclasses import dataclass
+from typing import Optional
+
+from rest_framework import serializers
+
+from ansible_ai_connect.ai.api.model_pipelines.config_pipelines import (
+    BaseConfig,
+    PipelineConfiguration,
+)
+from ansible_ai_connect.ai.api.model_pipelines.config_serializers import (
+    BaseConfigSerializer,
+)
+from ansible_ai_connect.ai.api.model_pipelines.registry import Register
+
+# ANSIBLE_AI_MODEL_MESH_API_URL
+# ANSIBLE_AI_MODEL_MESH_API_TIMEOUT
+# ANSIBLE_AI_MODEL_MESH_API_VERIFY_SSL
+
+
+@dataclass
+class LlamaCppConfiguration(BaseConfig):
+
+    def __init__(
+        self,
+        inference_url: str,
+        model_id: str,
+        timeout: Optional[int],
+        verify_ssl: bool,
+    ):
+        super().__init__(model_id)
```

Comment:
I do have a _serialiser_ for each _configuration_ class that perform validation.

For example: https://github.com/ansible/ansible-ai-connect-service/pull/1389/files#diff-27fc57b5bb6af1905a730ede9a62ecd3b390f5999d28b679cb8c3afa0748ce99R68-R72

This PR simply uses a JSON formatted environment variable; but it could equally come from a database or elsewhere. I'd prefer to handle a move to a database as another JIRA/PR as it affects concrete configuration in ASW Secret Manager a lot more.